backend/src/test/java/ch/puzzle/okr/controller/TeamControllerIT.java

```diff
@@ -101,4 +106,42 @@ void shouldGetAllTeamsIfNoTeamsExists() throws Exception {
                 .andExpect(jsonPath("$", Matchers.hasSize(0)))
         ;
     }
+
+    @Test
+    void shouldReturnTeamWhenCreatingNewTeam() throws Exception {
+        BDDMockito.given(teamService.saveTeam(any())).willReturn(teamTestCreating);
+
+        mvc.perform(post("/api/v1/teams")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content("{\"name\":\" TestTeam \"}"))
+                .andExpect(MockMvcResultMatchers.status().is2xxSuccessful())
+                .andExpect(MockMvcResultMatchers.content().string("{\"id\":1,\"name\":\"TestTeam\"}"))
+        ;
+        verify(teamService, times(1)).saveTeam(any());
+    }
+
+    @Test
+    void shouldReturnResponseStatusExceptionWhenCreatingTeamNullName() throws Exception {
+        BDDMockito.given(teamService.saveTeam(any()))
+                .willThrow(new ResponseStatusException(HttpStatus.BAD_REQUEST, "Missing attribute name when creating team"));
+
+        mvc.perform(post("/api/v1/teams")
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .content("{\"id\": 22, \"name\": null}"))
+                .andExpect(MockMvcResultMatchers.status().isBadRequest())
+        ;
+    }
+
+    @Test
+    void shouldReturnResponseStatusExceptionWhenCreatingTeamEmptyName() throws Exception {
```

Comment:
Grundsätzlich genügt in diesem Context 1 Test mit einer Exception. Wir prüfen hier, dass bei auftretender Exception die Information der Exception auch ausgegeben wird. Aktuall leider nur der Status. Ob die richtige Exception geworfen wird, wird der TeamServiceTest überprüfen.
Somit diese Testmethode löschen.